addon/components/o-s-s/carousel.ts

```diff
@@ -9,6 +9,7 @@ interface OSSCarouselArgs {
   buttonIcon?: string;
   animationStyle?: 'shift' | 'slide';
   showIndicators?: boolean;
+  showPageCounterBottom?: boolean;
```

Comment:
the argument just above allows to hide & display the `indicators` can we keep a similar naming scheme ?
-> `indicatorsPosition: 'top' | 'bottom'` defaults to where it currently is (top I think)

Reply:
Storybook & test files will need to be updated as well